src/marqo/core/embed/embed.py

```diff
@@ -67,6 +82,16 @@ def embed_content(
         else:
             raise base_exceptions.InternalError(f"Content type {type(content)} is not supported for embed endpoint.")
 
+        # Decide on the prefix 
+        if content_type == EmbedContentType.Query:
+            prefix = determine_text_prefix(None, marqo_index, "text_query_prefix")
+        elif content_type == EmbedContentType.Document:
+            prefix = determine_text_prefix(None, marqo_index, "text_chunk_prefix")
```

Comment:
Regarding this, I am removing `EmbedContentType` so we don't have to expose the `EmbedContentType` to users. We not take string, as before, but added a pydantic validator to limit the responses

```
@pydantic.validator('content_type', pre=True, always=True)
    def validate_content_type(cls, v):
        if v is not None and v not in ["query", "document"]:
            raise ValueError("content_type must be 'query', 'document', or None")
        return v
```

this can be found int `embed_request.py`